app/src/main/kotlin/com/metrolist/music/widget/PlaylistWidgetManager.kt

```
@@ -0,0 +1,630 @@
+    }
+
+    private suspend fun loadBitmap(uri: String, size: Int): Bitmap? = withContext(Dispatchers.IO) {
+        try {
+            val request = ImageRequest.Builder(context)
+                .data(uri)
+                .size(size, size)
+                .allowHardware(false)
+                .crossfade(false)
+                .build()
+            imageLoader.execute(request).image?.toBitmap()
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)